examples/notebook/hello-world-with-private-registry.ipynb

```diff
@@ -0,0 +1,195 @@
+      "INFO:root:Layer sha256:7babe47a4c402afbe26f10dffceb85be7bfd2072a96b816814503f41ce9c5273 exists, skipping\n",
+      "INFO:root:Layer sha256:5bac0c144f6e0b7082e3691da95d3f057ee0be0735e9efca76096da59cfd1786 exists, skipping\n",
+      "INFO:root:Layer sha256:21a7832aeb8625dc8228ceb115a28222f87e0fbce61b2588c42a2cce7a3a63d6 exists, skipping\n",
+      "INFO:root:Layer sha256:2b940936f9933b7737cf407f2149dd7393998d7a0bee5acf1c4a57b0487cef79 exists, skipping\n",
+      "INFO:root:Layer sha256:b3589acd4768bdc5c22cb66071dbf97a2948aeaf70c77eec64d6114aefab7230 exists, skipping\n",
+      "INFO:root:Layer sha256:d684674aa1a4d080be26286fd9356f573b80d2448599392e3dcf3c61ce98a0f0 exists, skipping\n",
+      "INFO:root:Layer sha256:823f4685c03b26a545ca41dcdca1e782ad5e52cf85bac03113edaa6aebdca1b3 exists, skipping\n",
+      "INFO:root:Layer sha256:e3ab47ad84d9e11c5fad45791ce00ec5b5f3b7f1ae61a5fab17eb44c399d910f exists, skipping\n",
+      "INFO:root:Layer sha256:330a9002e0b4aa1e27d3628dd3f02ff9a39d25745b8f2f219b06e3725153ffc0 exists, skipping\n",
+      "INFO:root:Layer sha256:14ca88e9f6723ce82bc14b241cda8634f6d19677184691d086662641ab96fe68 exists, skipping\n",
+      "INFO:root:Layer sha256:4e8a6b90828e0d339f646d723df8720ffa17c0ffb905f8f009faf1be320ab5d9 exists, skipping\n",
+      "INFO:root:Layer sha256:865f66ed1041c22ec4bca64bc28e446bf74bd8d77c5b8d9d169ebf7a3ef78115 pushed.\n",
+      "INFO:root:Layer sha256:92d24c89f5bc70958385728755b042a5a45bddf2f997de80e84d1161f43ba316 exists, skipping\n",
+      "INFO:root:Layer sha256:dc2840b4417186d66a29d64a039ac164be95929211d808294d36acae9301fc6b exists, skipping\n",
+      "INFO:root:Layer sha256:35daced67e5901b8de4a92bca9fdc67c8593d400aae483591987442f54c87d0a exists, skipping\n",
+      "INFO:root:Layer sha256:5bd1cb59702536c10e96bb14e54846922c9b257580d4e2c733076a922525240b exists, skipping\n",
+      "INFO:root:Finished upload of: registry.zipsacoding.com/fairing-job:4DC5EA4\n",
+      "WARNING:kubeflow.fairing.builders.append.append:Pushed image registry.zipsacoding.com/fairing-job:4DC5EA4 in 0.48132142299436964s.\n",
+      "WARNING:kubeflow.fairing.deployers.job.job:The job fairing-job-x8qqq launched.\n",
+      "WARNING:kubeflow.fairing.kubernetes.manager:Waiting for fairing-job-x8qqq-wwrf9 to start...\n",
+      "WARNING:kubeflow.fairing.kubernetes.manager:Waiting for fairing-job-x8qqq-wwrf9 to start...\n",
+      "WARNING:kubeflow.fairing.kubernetes.manager:Waiting for fairing-job-x8qqq-wwrf9 to start...\n",
+      "WARNING:kubeflow.fairing.kubernetes.manager:Waiting for fairing-job-x8qqq-wwrf9 to start...\n"
+     ]
+    },
+    {
+     "ename": "KeyboardInterrupt",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
```

Comment:
There is traceback? do not interrupt or clear outputs?

Reply:
I don't know why "output" is included. I'll fix it.